x-pack/plugins/triggers_actions_ui/public/application/sections/alerts_list/components/alerts_list.scss

```
@@ -5,3 +5,29 @@
     color: $euiColorDarkShade;
   }
 }
+
+.alertSidebarItem {
```

Should this be on hover of the whole row where we display the edit / delete icons? cc @mdefazio 

```suggestion
.euiTableRow {
```